static/js/src/tabotronic.js

```diff
@@ -47,4 +47,13 @@
   window.addEventListener("hashchange", function () {
     hashChange();
   });
+
+  const selectedTabGroup = document.querySelector("[data-js-selected-tab]");
+  if (selectedTabGroup) {
+    const selectedTab = selectedTabGroup.querySelector(
+      "#" + selectedTabGroup.dataset.jsSelectedTab
+    );
+    const selectedTabId = selectedTab.getAttribute("aria-controls");
+    tabClickHandler(selectedTab, selectedTabId);
```

Comment:
I think this is what's scrolling the page on load - there is an element with `[data-js-selected-tab]` present when you load the page, and `tabClickHandler` scrolls the element into view (line 36).

Reply:
An alternative to tabotronic is https://github.com/canonical-web-and-design/canonical.com/blob/master/static/js/tabs.js - it allows switching of tabs without causing the page to jump around.